kornia/geometry/epipolar/fundamental.py

```diff
@@ -70,9 +70,212 @@ def normalize_transformation(M: Tensor, eps: float = 1e-8) -> Tensor:
     return torch.where(norm_val.abs() > eps, M / (norm_val + eps), M)
 
 
-def find_fundamental(
-    points1: torch.Tensor, points2: torch.Tensor, weights: Optional[torch.Tensor] = None
-) -> torch.Tensor:
+# Reference : https://github.com/opencv/opencv/blob/4.x/modules/calib3d/src/polynom_solver.cpp
+def solve_quadratic(a: torch.Tensor, b: torch.Tensor, c: torch.Tensor) -> torch.Tensor:
```

Comment:
`from kornia.core import Tensor` -- can you update all the Tensor typing in this file, please? 
```suggestion
def solve_quadratic(a: Tensor, b: Tensor, c: Tensor) -> Tensor:
```